propolis/src/block.rs

```diff
@@ -149,6 +167,9 @@ impl<R: BlockReq> PlainBdev<R> {
         }
         BlockResult::Success
     }
+
+    /// Spawns a new thread named `name` on the dispatcher `disp` which
+    /// begins processing incoming requests.
```

Comment:
"incoming requests" is a little too specific.  That thread can do whatever it wants